common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -801,6 +876,13 @@ public void run() {
         });
     }
 
+    /**
+     * An interface that is called with a long argument.
+     */
+    protected interface LongCallable {
```

Comment:
Because we can't use java 8 we have the concept of `Supplier` (e.g. `io.netty.util.BooleanSupplier`). Can we just make this a `io.netty.util.LongConsumer`? It should make it easier to find / cleanup when we decide to drop older JDKs.